Sources/AppStoreConnectCLI/Helpers/File+Helper.swift

```diff
@@ -0,0 +1,12 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import Foundation
+import Files
+
+extension File {
+    static func folderAndFilename(from path: String) -> (String, String) {
+        let filePath: NSString = path as NSString
+        
+        return (filePath.deletingLastPathComponent, filePath.lastPathComponent)
```

Comment:
You can name your tuple arguments here for increased readability

```suggestion
    static func folderAndFilename(from path: String) -> (folderName: String, fileName: String) {
        let filePath: NSString = path as NSString
        
        return (folderName: filePath.deletingLastPathComponent, fileName: filePath.lastPathComponent)
```